client-vms/src/types/values-permissions.ts

```diff
@@ -136,6 +143,14 @@ export class ValuesPermissionsBuilder {
     );
   }
 
+  static default_owner(owner: UserId): ValuesPermissionsBuilder {
```

Comment:
nit: worth sticking with camelCase and PascalCase in the JS/TS ecosystem.

Reply:
right, I will fix it before the merge